src/infrastructure/database/utils/type-conversion.ts

```
@@ -151,35 +162,49 @@ export function convertTimestamp(
+          type: typeof value,
+          value,
+        });
         return null;
       }
-    } else { // dbType === 'sqlite'
+    } else {
+      // dbType === 'sqlite'
       // Assume SQLite returns a number (epoch ms)
       if (typeof value === 'number') {
         // Check for non-finite numbers before creating Date
         if (!isFinite(value)) {
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)